app/assets/stylesheets/_overlay.scss

```diff
@@ -77,7 +77,14 @@ body > .fullscreen-overlay {
       }
     }
   }
-
+  .ui-icon-x-thin {
```

Comment:
scope this to something else, we might need to use that icon for other things within an overlay